docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -31,19 +31,12 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
-  - **LIKE**
-
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+
+  The last operator (**ORDER BY**) is supported only with Elasticsearch as a Visibility store.
+  Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+- To search for a specific word, set a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` to a phrase that contains that word.
+  Any command run with that Search Attribute returns any Workflows that contain that word.
```

Comment:
```suggestion
- To use a partial string match (i.e. specific word), create a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` and set it value to a phrase that contains that word. Use equal operator `=` in List Filters to find Workflows which have Search Attribute that contain that word.
```

And I do believe that good example will really help here. Something like:
===
If `Description` Search Attribute of type `Text` is set to "The quick brown fox jumps over the lazy dog" then List Filters like `Description="quick"` or `Description="fox"` will return this workflow. Note that it works only for full word match. List Filters like `Description="qui"` or `Description="laz"` won't return this workflow. Read more about standard Elasticsearch tokenizer at https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html
===